test/e2e/main_test.go

```diff
@@ -73,17 +73,12 @@ func TestMain(m *testing.M) {
 	flag.StringVar(&testOutputPath, "test-output-path", "", "The absolute/relative path that you want to store test logs within.")
 	flag.Parse()
 
-	logger := testhelpers.SetupLogger(logLevel)
+	logger = testhelpers.SetupLogger(logLevel)
 
```

Comment:
Looks good so far. I think that because this is in a function the short assign would be an option too

Reply:
I think if I create and assign it here then it wouldn't be accessible in the other places it's used?